playbooks/scenario2c_coreos_az.yaml

```diff
@@ -0,0 +1,111 @@
+---
+# Scenario to test ECS service with CoreOS image in different AZs
+# A VPC is created and VM is being started in different AZs
+#
+
+- name: Scenario 2c - Compute per AZ
+  hosts: localhost
+  vars:
+    prefix: scenario2c-
+    test_image_url: https://builds.coreos.fedoraproject.org/prod/streams/stable/builds/33.20210117.3.2/x86_64/fedora-coreos-33.20210117.3.2-openstack.x86_64.qcow2.xz
```

Comment:
I somehow dislike the hardcoding of specific image url
CoreOS is offering several universal ways how to fetch the latest image:
Use stable.json for getting the latest image location:

```
 curl -sS https://builds.coreos.fedoraproject.org/streams/stable.json | jq '.architectures.x86_64.artifacts.openstack.formats' | jq -r .'["qcow2.xz"].disk.location'
https://builds.coreos.fedoraproject.org/prod/streams/stable/builds/33.20210117.3.2/x86_64/fedora-coreos-33.20210117.3.2-openstack.x86_64.qcow2.xz
```

or use coreos-installer but would then really download the image locally (which we probably don't want)

`coreos-installer download --stream stable --platform openstack --decompress  --format qcow2.xz`

Reply:
My mostly disliked thing around CoreOS is that it changes too fast and too unpredictable. I really want to hardcode specific version of the image to have reproducible results